dbt_utils_integration_tests_howto/integration_tests__models__sql__schema__yml.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/integration_tests/models/sql/schema.yml b/integration_tests/models/sql/schema.yml
+index e79e782..5258a39 100644
+--- a/integration_tests/models/sql/schema.yml
++++ b/integration_tests/models/sql/schema.yml
+@@ -213,11 +213,6 @@ models:
+       - name: favorite_number
+         tests:
+           - dbt_utils.not_constant
+-  
+-  - name: test_union_no_source_column
+-    tests:
+-      - expect_table_columns_to_match_set:
+-          column_list: ["id", "name", "favorite_color", "favorite_number"]
+ 
+   - name: test_union_exclude_lowercase
+     tests:
+@@ -258,6 +253,9 @@ models:
+         tests:
+           - dbt_utils.not_empty_string:
+               trim_whitespace: false
++              config:
++                error_if: "!=1"
++                warn_if: "<0"
```

Comment:
Can you please explain this configuration?
And the reason for removing "test_union_no_source_column" tests.

Reply:
teradata treats a value that is made of multiple blanks as '', in the test data there is one such string , and this failure captures this.

Reply:
see line 47 in dbt_utils_integration_tests_howto/integration_tests__models__sql__test_not_empty_string_passing__sql.patch

Reply:
Got it. Thanks